dockerfiles/centos/is/Dockerfile

```diff
@@ -37,8 +37,7 @@ ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}-${WSO2_SERVER_VERSION}
 
 RUN yum check-update && \
-    yum install --downloadonly \
-    nc && \
+    yum install -y  nc && \
```

Comment:
what does -y correspond in here? also remove the additional space in between -y and nc.